src/client/components/MealListComponent.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useEffect, useState } from "react";
-import MealListItem from "./MealListItem";
+import MealItem from "./MealItem";
```

Comment:
Can't comment on code that has not been changed in this PR, but fetch seems good 👍

Reply:
Yeap, it works :) 
Thank you for reviews, btw!